protocol/bin/flow.js

```diff
@@ -370,9 +362,8 @@ import type {Boolean, Bool, Bytes, Double, Int, Int64, Long, String, Uint, Uint6
     Object.keys(project.incomingMaps).map(im => `  '${im}'?: ${project.incomingMaps[im]}`).join(',') +
     '|}\n'
   const toWrite = [typePrelude, typeDefs.join('\n'), incomingMap].join('\n')
-  // const formatted = prettier.format(toWrite, prettierOptions)
-  // TODO disabling prettier short term
-  fs.writeFileSync(project.out, /*formatted*/ toWrite)
+  const formatted = prettier.format(toWrite, prettier.resolveConfig.sync(project.out))
```

Comment:
@patrickxb can you verify this branch works. i think we have to remove this cause you couldn't run prettier effectively

Reply:
works, thanks